src/makeRobot.js

```diff
@@ -37,8 +37,90 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    goForward(value = 0) {
+      if (value < 0) {
+        return this; ;
+      } else if (value === 0) {
+        this.coords.y++;
+
+        return this;
+      } else {
+        this.coords.y = this.coords.y + value;
+
+        return this;
+      }
+    },
```

Comment:
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
    },
```

just set default value 1 and you don't need these conditionals

Reply:
fix all such cases